server/requirements.txt

```diff
@@ -40,6 +40,6 @@ python3-ldap==0.9.5.3
 behave==1.2.5
 wooper==0.4.2
 
-git+git://github.com/superdesk/eve@sync-upstream#egg=Eve==0.6-dev
```

Comment:
how is enough just e49cca when the full hash is e49cca9e48b4d33fe362305d8463314113642a6f?